controllers/common/common.go

```diff
@@ -286,7 +286,7 @@ func (h *ErrorHandler) HandleReconcilerError(request reconcile.Request, in error
 		result = RetryTransientError
 		err = nil
 
-		h.Info("waiting for dependency status", "request", request)
```

Comment:
Is the error you want to handle is in a transient status? If not, please define another type of error instead